src/goods.ts

```diff
@@ -53,19 +53,16 @@ export async function fetch(url: RequestInfo, init?: RequestInit) {
 }
 
 // A console.log() alternative which can take ProcessOutput.
-export function echo(
-  pieces: TemplateStringsArray | ProcessOutput | string,
-  ...args: any[]
-) {
+export function echo(pieces: any, ...args: any[]) {
```

Comment:
This is not valid again, as for str template, the `TemplateStringsArray` is preferable.